src/comps/update/index.js

```diff
@@ -1,17 +1,27 @@
 const db = require("../../db")
 
 async function cleanDb(dbname, round) {
+    if (!round !== undefined) {
+        await db.promise().query(`DELETE FROM ${dbname}`)
```

Comment:
this looks dangerous, but rest lgtm

Reply:
```!round !== undefined```

I see the **!**round being a bug (the exclamation mark)

But, is there another reason why you say "this looks dangerous"?